deposit/src/main/java/edu/unc/lib/deposit/fcrepo4/IngestContentObjectsJob.java

```diff
@@ -396,6 +396,15 @@ private FileObject addFileToWork(WorkObject work, Resource childResc)
         // Retry if there are checksum failures
         for (int retryCnt = 1; retryCnt <= CHECKSUM_RETRIES; retryCnt++) {
             try {
+                if (retryCnt > 1) {
```

Comment:
might be simpler to perform the delay in the `ChecksumMismatchException` catch, it already has a block to check if retries are remaining and the automatic result would be there wouldn't be a delay until after the first try.